packages/effect/src/Effect.ts

```diff
@@ -5392,3 +5392,27 @@ export const Tag: <const Id extends string>(id: Id) => <Self, Type>() =>
     })
     return done
   }
+
+/**
+ * Creates an `AbortSignal`.
+ * @since 3.1.0
+ */
+export const makeAbortSignal: Effect<AbortSignal> = map(
+  tap(
+    sync(() => new AbortController()),
+    (controller) => onInterrupt(() => sync(() => controller.abort()))
```

Comment:
This doesn't make sense, as the onInterrupt effect is only applied regionally / to the effect it wraps.

Reply:
Oh interesting. Ok. I'll remove this.